graphql/resolvers/index.js

```diff
@@ -82,6 +82,29 @@ const login = async (type, email, password) => {
   };
 };
 
+/**
+ * Get a comparator function which specifies the order of two objects depending on specific field (key).
+ * This compare function then can be passed to Array.sort(comparator) to sort an array of objects by the key.
+ * 
+ * @param key The key on which the returned function should compare the objects
+ * @returns   Function which returns -1, 0 or 1 based on the order relation 
+ *            of the corresponding field of the objects, specified by key
+ */
+const getDishComparator = key => {
+  return (dish1, dish2) => {
+    let key1 = dish1[key];
```

Comment:
you can make these constants

```suggestion
    const key1 = dish1[key];
```

Reply:
I created these variables so that I'm able to modify them later: 
```
if(key1 + '' === key1) {
      key1 = key1.toLowerCase(); 
      key2 = key2.toLowerCase();
}
```

If they were constants, that wouldn't have been possible.